src/cli/commands/dag/get.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
       // * reads as 'agree in'
       if (node._json) {
         delete node._json.multihash
-        node._json.data = '0x' + node._json.data.toString('hex')
+        node._json.data = node._json.data.toString('base64')
```

Comment:
Is this change meant to be in this PR? Why not also for the if test below?

Reply:
Probably could be separate.  go prints out the data as base64 encoded, we print it out as hex.